gui/src/utils.js

```diff
@@ -29,3 +29,13 @@ export const startRecipe = name => {
     .then(response => response.json())
     .then(data => console.log(data))
 }
+
+export const uploadRecipe = file => {
+  const formData = new FormData();
+  formData.append("File", file);
+  fetch(apiUrl + 'uploadRecipe', {
+    method: 'POST',
+    body: formData,
+  }).then(response => response.json())
+    .then(data => console.log(data))
```

Comment:
Should we have better feedback here?  Or is that feedback a WIP?

Reply:
Currently I don't think there's any feedback to the user if any request fails so I think that's just a separate thing that needs to be added.

Reply:
For success not 100% sure the best way to handle feedback, I was just going to make it so at least it would get rid of displaying the filename you just uploaded if the upload was successful for at least some feedback, but apparently file inputs are forced to be uncontrolled inputs by react so that's not something we can do.